mirdata/datasets/jtd.py

```diff
@@ -0,0 +1,722 @@
+"""Jazz Trio Database (JTD) Loader
+
+.. admonition:: Dataset Info
+    :class: dropdown
+
+    The Jazz Trio Database (JTD) is a dataset comprising 1,294 multitrack jazz performances (about 45 hours total)
+    annotated by an automated signal processing pipeline. All performances are commercial recordings of jazz piano
+    trios, comprising acoustic piano, upright bass, and drum kit, and are broadly in the "straight-ahead" jazz style.
+
+    Its purpose is to serve as a reference database for the design, evaluation, and implementation of various music
+    information retrieval systems related to jazz and improvised music, including (but not limited to) onset detection,
+    beat tracking, automatic music transcription, and performer identification.
+
+    For every performance, the following audio files are included:
+
+    1) the "raw" audio from the piano solo in the performance (stereo, 44.1 kHz)
```

Comment:
maybe add "typically including piano, bass, and drums"? Otherwise it sounds a bit like the audio might be piano only